jwql/instrument_monitors/common_monitors/bad_pixel_monitor.py

```diff
@@ -1059,9 +1059,9 @@ def process(self, illuminated_raw_files, illuminated_slope_files, flat_file_coun
         if 'NOT FOUND' in baseline_file:
             logging.warning(('\tNo baseline bad pixel file for {} {}. Any bad '
                              'pixels found as part of this search will be considered new'.format(self.instrument, self.aperture)))
-            baseline_file = new_badpix_file
-            yd, xd = badpix_mask.shape
-            baseline_badpix_mask = np.zeros((yd, xd), type=np.int)
+            baseline_file = 'None'
```

Comment:
Curious if this should be a string or `None`.  If you haven't already done so, please examine `get_baseline_filename(self)` and verify that this string is appropriate.

Reply:
The value of baseline_filename ends up being saved in the results database table, so I believe a string is correct.

Reply:
Sounds good to me. I know `None` python types don't play well with databases.